imcsdk/apis/admin/user.py

```diff
@@ -118,6 +118,10 @@ def password_expiration_exists(handle, **kwargs):
     if mo is None:
         return False, None
 
+    for k, v in kwargs.items():
+        if isinstance(v, int):
+            kwargs[k] = str(v)
+
```

Comment:
@waghswapnil this looks fishy!

so is this specific to this MO? that all properties are strings? But the user might specify some as ints?
What is the type in the MO?